setup.py

```diff
@@ -29,9 +29,10 @@
         'python-slugify'
     ],
     tests_require=[
+        'nose>=1.3.7',
         'mock>=1.0.1',
         'testfixtures>=4.1.2',
-        'salt==2014.7.5',
+        'salt==2015.5.11',
```

Comment:
Sorry, I'll update the commit message. I didn't mention about the other setup.py changes either. Give me a few minutes, just trying to get the tests running in travis.